src/app/api/comments/likes/route.ts

```diff
@@ -0,0 +1,88 @@
+import prisma from "@/lib/prisma";
+import { NextResponse, type NextRequest } from "next/server";
+
+export async function POST(req: NextRequest) {
+  try {
+    const url = new URL(req.url);
+    const commentId = url.searchParams.get("commentId");
+    const userId = url.searchParams.get("userId");
+
+    if (!commentId || !userId) {
+      return new NextResponse(
+        JSON.stringify({ error: "commentId and userId are required" }),
+        {
+          status: 400,
+          headers: { "Content-Type": "application/json" },
+        },
+      );
+    }
+
+    const like = await prisma.commentLike.create({
+      data: {
+        commentId: commentId,
+        userId: userId,
+      },
+    });
+
+    return new NextResponse(
+      JSON.stringify({ message: "Comment liked successfully", like }),
+      {
+        status: 201,
+        headers: { "Content-Type": "application/json" },
+      },
+    );
+  } catch (error) {
```

Comment:
it looks like this might be the catch that includes the error that occurs when the like already exists.  Should we check if the like exists, first, so we can have a more specific error for the front end to work with?